dashboard/config/database.yml

```diff
@@ -1,17 +1,6 @@
-<%
-# TODO: Rails 4.1 may be able to be configured with the database url
-# instead of having to parse it here. Actually 4.0 can do that too but
-# it doesn't work with the way seamless_database_pool wants to
-# configure the read pool
-
-writer = URI.parse(CDO.dashboard_db_writer) if CDO.dashboard_db_writer
-reader = URI.parse(CDO.dashboard_db_reader) if CDO.dashboard_db_reader
-%>
-
 mysql_defaults: &mysql_defaults
-  adapter: seamless_database_pool
+  adapter: mysql2
   prepared_statements: false
```

Comment:
Not necessary for this PR, but just noting that we might be able to re-enable `prepared_statements` after removing `seamless_database_pool`, I think we originally disabled this due to some incompatibility with that gem.